### PR TITLE
[MLU]transpose convbpf output to HWCN for better performance

### PR DESCRIPTION
**Pull request by Ashelly1**
### PR types

Performance optimization
### PR changes

OPs
### Describe

transpose convbpf output to HWCN for better performance

### CODE REVIEW

**Comment by cambriconhsq** [comment]
:+1:

File: paddle/fluid/operators/conv_op_mlu.cc [hunk 1]
```diff
@@ -462,10 +463,12 @@ class MLUDepthwiseConvGradOpKernel : public framework::OpKernel<T> {
 
       auto filter_grad_dims = filter_grad->dims();
       Tensor temp_filter_grad(filter_grad->type());
-      temp_filter_grad.mutable_data<T>({filter_grad_dims[0],
-                                        filter_grad_dims[2],
+      // Details about setting diff_w hwcn for better performance, see the CNNL
+      // documentation.
+      temp_filter_grad.mutable_data<T>({filter_grad_dims[2],
```
Comment (ShawnNew) [inline paddle/fluid/operators/conv_op_mlu.cc]: 这里的index属于magic number，你可以用上面的perm_xxx_to_xxx vector定义的index顺序

**Review by ShawnNew — CHANGES_REQUESTED** [review]

File: paddle/fluid/operators/conv_op_mlu.cc (hunk 1 above)
Comment (Ashelly1) [reply paddle/fluid/operators/conv_op_mlu.cc]: 已修改

**Review by ShawnNew — APPROVED** [review]
👍

**Review by ronny1996 — APPROVED** [review]
LGTM

**Pull request merged**